snuba/processor.py

```diff
@@ -53,13 +55,18 @@ def _as_dict_safe(value):
     return rv
 
 
-def _collapse_uint32(n):
-    if (n is None) or (n < 0) or (n > MAX_UINT32):
+def _collapse_uint32(n) -> Optional[int]:
```

Comment:
I intentionally left the input parameter blank (not `Any`) — I'm not sure it makes sense to accept an explicit `Any` since there is a finite set of parameters that can be converted to `int`. I just didn't take the time to figure out what is in that set.